### DISCUSSION TITLE
When opening a feature request issue, what is the difference between "Requirements" and "Acceptance Criteria"

### DISCUSSION THREAD

**Discussion by mkavulich** [ufs-community/ufs-srweather-app#585] [Q&A]
It seems to me that for every use case I can think of, the "acceptance criteria" is that the "requirements" are met. Is there some meaningful difference that I am missing? Or can these two sections be consolidated?

Additionally, it seems like the "Requirements**" section have a double-asterisk that doesn't point to anything, can this be removed?

**Comment by christinaholtNOAA** [comment]
Google says, "Requirements are what you're supposed to do. Acceptance Criteria are the agreed upon measures to prove you've done them."

Here's another interesting read: https://www.techtarget.com/searchsoftwarequality/answer/How-do-requirements-relate-to-acceptance-criteria

**Comment by gspetro-NOAA** [comment]
@mkavulich I think the asterisks can be removed. My guess is that I copy-pasted some bolded text and changed it to a heading (with ##) rather than bold (with **) but forgot to remove the asterisks on both sides. 
As to requirements versus acceptance criteria, I agree that there's a lot of overlap, but I don't think they're the same. For example, when I need data added to Level 1 platforms, the requirements section should list the specific file names and platforms (ideally). The acceptance criteria would be that the data (for XYZ task - e.g., the tutorial chapter) appears on Level 1 platforms. Without the requirements, it's difficult to know whether the data that has been added is enough to meet the acceptance criteria. I suppose that in theory, we could get rid of "acceptance criteria" since it's sort of a summary of the requirements. 
In cases where the requirements and acceptance criteria are pretty similar, it's not hard to copy-paste the same text, but I do think there may be times where they differ, and it is useful to have both. That said, I'm not too bothered if we decided they are redundant enough to remove.

**Reply by mkavulich** [reply]
Thanks for the reply. I'm fine leaving the two sections if some people think they may be useful, I just don't want people to think I'm being flippant when I'm just writing "All requirements are met" in my Acceptance criteria section every time 😊
